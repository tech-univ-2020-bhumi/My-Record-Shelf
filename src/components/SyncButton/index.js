/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import propTypes from 'prop-types';

function SyncButton(props) {
  const { testId } = props;
  return (
    <button type="button" data-testid={testId}>sync</button>
  );
}

SyncButton.propTypes = {
  testId: propTypes.func.isRequired,
};

export default SyncButton;
