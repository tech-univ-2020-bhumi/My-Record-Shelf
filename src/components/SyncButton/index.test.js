/* eslint-disable no-undef */
/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import SyncButton from './index';

describe('The SyncButton component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SyncButton testId="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
