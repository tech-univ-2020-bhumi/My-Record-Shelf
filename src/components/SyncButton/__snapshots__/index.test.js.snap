// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The SyncButton component should render correctly 1`] = `
<DocumentFragment>
  <button
    data-testid="123"
    type="button"
  >
    sync
  </button>
</DocumentFragment>
`;
